README.md

```diff
@@ -1,8 +1,15 @@
+## :warning: Deprecation warning
+In preparation for the KiCad v6 release this repository will be locked down on Sept 1, 2020. No new pull-requests will be accepted. Existing pull-requests can be worked on until Oct 1, 2020. Changes breaking v5.1 compatibility can be merged starting Sept 1, 2020.
+
+On 2020-10-01 this repository, including issues and pull requests, will be archived and transferred to [gitlab.com](https://gitlab.com/kicad/libraries/kicad-symbols/). In order for the GitLab importer to function properly it is recommended you set your email address on GitHub to public and use the same address for your GitLab account. Or login to GitLab at least once using the GitHub icon. Otherwise the importer can't correlate the account information and the issues/comments on GitLab will be owned by `kicad-bot` ([importer documentation](https://docs.gitlab.com/ee/user/project/import/github.html#how-it-works)).
+
+We plan to convert the library to the new v6 S-expr format after it is imported to GitLab. That also means that old pull requests will need to be redone with v6 format if they are not merged before then.
+
 # KiCad Symbols
 
 This repository contains the official KiCad schematic symbol libraries.
 
-**The libraries in this repositiory are intended for KiCad version 5 or a nightly build that supports the schematic library version 2.4 or newer.**
+**The libraries in this repositiory are intended for KiCad version 5**
```

Comment:
Why not make this "5.x" for clarity? In fact, the old text seems valid to me if you just remove the final "or newer".

Reply:
The original text meant 4.99 nightlies. Since current nightlies use the S-expr format this is misleading. `5.x' is a good suggestion.

Reply:
Funny side-node, quite a few libs in current master have a `EESchema-LIBRARY Version 2.3` string as their first line.